src/copy.jl

```diff
@@ -137,3 +137,15 @@ function Base.copy(model::AbstractModel)
     new_model, _ = copy_model(model)
     return new_model
 end
+
+"""
+    deepcopy(_::AbstractModel)
+
+Calls error. Calling `deepcopy` over a JuMP model is not supported, nor
+planned to be supported, because it would involve making a deep copy of
+the underlying solver (behind a C pointer). Check [`copy`](@ref) instead.
+"""
+function Base.deepcopy(_::AbstractModel)
```

Comment:
`Base.deepcopy(::AbstractModel)` instead of `Base.deepcopy(_::AbstractModel)`

Reply:
Done.